sonar-plugin-api/src/main/java/org/sonar/api/utils/StaxParser.java

```diff
@@ -37,7 +37,9 @@
 
 /**
  * @since 1.10
+ * @deprecated since 5.6 plugins should use their own dependencies
  */
+@Deprecated
 public class StaxParser {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)